python/cuml/cuml/tests/test_fil.py

```diff
@@ -21,6 +21,8 @@
 import pytest
 import treelite
 
+# Import XGBoost before scikit-learn to work around a libgomp bug
+# See https://github.com/rapidsai/cuml/pull/6728#issuecomment-2899839082
```

Comment:
Can you add a note to the docs for this?

Reply:
I updated the comment to point to https://github.com/dmlc/xgboost/issues/7110, which has a better description of what's going on with libgomp.

Reply:
This piece of code is part of the test module (`cuml.tests`). To my knowledge, we do not import XGBoost inside cuML itself. So to trigger the bug, the user will have to do the following:
1. The user uses an arm64 machine.
2. The user installs cuML, XGBoost, and scikit-learn from pip wheels (not Conda).
3. The user imports scikit-learn, and then `cuml.tests`.